src/Surfnet/StepupMiddleware/ApiBundle/Identity/Service/AbstractSearchService.php

```diff
@@ -54,4 +54,26 @@ protected function createPaginatorFrom($doctrineQuery, AbstractQuery $query)
 
         return $paginator;
     }
+
+    /**
+     * @param Query $doctrineQuery
+     * @return array
+     */
+    protected function getFilteredQueryOptions(Query $doctrineQuery)
```

Comment:
In my opinion, moving this to the abstract search service is not the best solution. Yes it solves code duplication, but abstraction is not the preferred solution to solve this problem.

Reply:
I will leave it for now as-is because it's not a major concern. But feel free to address this later on.